src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -0,0 +1,68 @@
+import { classMethod } from "@babel/types";
```

Comment:
We should re-use [EmptyLines](https://cs.github.com/RecordReplay/devtools/blob/41b1d6b26a837dc4d78f73fa8346bbb001fc5b63/src/devtools/client/debugger/src/components/Editor/EmptyLines.js#L12) which is optimized for this

Reply:
I don't understand this comment.

Reply:
I think, from glancing at this a few weeks ago, that there's an `EmptyLines` component we have in the codebase that already has some skeleton handling for "loop over lines in the editor, skip ones that are empty, and apply some kind of behavior to the lines with content".  But didn't look further than that.